### PR TITLE
Adds sinew restraints, changes sinew droprate.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
Request by @KorPhaeron 

Watchers now drop two sinews, you can use sinew in hand to craft restraints. 

:cl: 
rscadd: Sinew can now be fashioned into restraints.
/:cl:

### CODE REVIEW

**Comment by Core0verload** [comment]
They should be better than cable cuffs, given how easy it is to get cable and how hard it is to get sinews.

**Comment by TechnoAlchemisto** [comment]
It's just skin but perhaps you are right, i'll just do whatever Kor says on this one to be honest.

**Comment by Lumipharon** [comment]
uncommon shit doesn't neccessarily need to be better shit simply because it's harder to get.

File: code/game/objects/items/stacks/sheets/leather.dm [hunk 1]
```diff
@@ -135,6 +135,14 @@ var/global/list/datum/stack_recipe/xeno_recipes = list ( \
...
 	origin_tech = "biotech=4"
 
+
+var/global/list/datum/stack_recipe/sinew_recipes = list ( \
+	new/datum/stack_recipe("sinew restraints", /obj/item/weapon/restraints/handcuffs/sinew, 1, on_floor = 1), \
+	)
+
+/obj/item/stack/sheet/sinew/New(var/loc, var/amount=null)
+	recipes = sinew_recipes
+	return ..()
```
Comment (duncathan) [inline code/game/objects/items/stacks/sheets/leather.dm]: Does anyone understand why this is the typical method for adding stack recipes

Comment (TechnoAlchemisto) [inline code/game/objects/items/stacks/sheets/leather.dm]: No because I just copy and paste my code

**Pull request merged**